docker-compose.zaakbrug.dev.yml

```diff
@@ -13,6 +13,15 @@ services:
     depends_on:
       - zaakbrug
 
+  postgres:
```

Comment:
It needs a volume for the data to persist when the Postgres container is rebuilt. There is already a volume defined in the docker-compose for this called: "database-data". For an example you can look at one of the other docker-compose files.